cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,46 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Settings page', () => {
-  before(() => {
+import SettingsPageObject from '../support/pages/settings.pageObject';
 
-  });
-
-  beforeEach(() => {
+beforeEach(() => {
+  const settingsPage = new SettingsPageObject();
 
+  describe('Settings page', () => {
+    cy.task('db:clear');
   });
 
   it('should provide an ability to update username', () => {
+    settingsPage.visit();
+    settingsPage.updateUsername('NewUsername');
```

Comment:
Ensure that the `updateUsername` method in `SettingsPageObject` is correctly implemented to update the username to 'NewUsername'.